docs/integrations/destinations/postgres.md

```diff
@@ -267,6 +267,7 @@ _where_ it is deployed.
 
 | Version | Date       | Pull Request                                               | Subject                                                                                                  |
 |:--------|:-----------|:-----------------------------------------------------------|:---------------------------------------------------------------------------------------------------------|
+| 2.1.0   | 2024-07-16 | [\#41954](https://github.com/airbytehq/airbyte/pull/41954) | add support for 1.0 refreshes.                                                          |
```

Comment:
nit: copy the changelog from https://github.com/airbytehq/airbyte/blob/master/docs/integrations/destinations/bigquery.md?plain=1#L232

Reply:
sorry, was thinking specifically of this one
> `Support for [refreshes](../../operator-guides/refreshes.md) and resumable full refresh. WARNING: You must upgrade to platform 0.63.7 before upgrading to this connector version.`

(i.e. telling people they need to  upgrade platform to get the new features + link to the operator guide)